addons/im_livechat/tests/test_user_livechat_username.py

```diff
@@ -81,3 +81,34 @@ def test_user_livechat_username_reactions(self):
                 "write_date": fields.Datetime.to_string(john.partner_id.write_date),
             },
         )
+
+    def test_user_livechat_username_in_store(self):
+        dog_expert = self.env["im_livechat.expertise"].create({"name": "dog"})
+        john = self._create_operator("fr_FR", expertises=dog_expert)
+        operator = self._create_operator("en_US")
+
+        john.livechat_username = "ELOPERADOR"
+        operator.group_ids = [
+            fields.Command.link(self.env.ref("im_livechat.im_livechat_group_user").id),
+            fields.Command.link(self.env.ref("base.group_user").id),
+        ]
+
+        livechat_channel = self.env['im_livechat.channel'].create({
+            'name': 'The channel',
+            'user_ids': [(6, 0, [john.id, operator.id])],
```

Comment:
Would be best to use commands instead of numbers.

Also apply formatter/double quotes.